src/audio/CMakeLists.txt

```diff
@@ -32,6 +32,12 @@ if(NOT BUILD_HOST)
 			src_hifi3.c
 		)
 	endif()
+	if(CONFIG_COMP_SEL)
```

Comment:
You also need to add it to Kconfig file.

Reply:
Thx, fixed